src/utils/crypto.ts

```diff
@@ -0,0 +1,26 @@
+import CryptoJS from 'crypto-js';
+import { Credential } from '../types';
+
+export function deCrypt(credential: Credential, key: string): Credential {
```

Comment:
What are you decrypting this is not a sufficient name
```suggestion
export function decrypt(credential: Credential, key: string): Credential {
```